From be117a342582326c4b88607331a39cf4cfe4d7c4 Mon Sep 17 00:00:00 2001
From: Aryan Sahoo <sahooaryan499@gmail.com>
Date: Wed, 31 Jan 2024 12:43:19 +0530
Subject: [PATCH 1/2] adding helloworld2.c to test branch

---
 hello_world2.c | 6 ++++++
 1 file changed, 6 insertions(+)
 create mode 100644 hello_world2.c

diff --git a/hello_world2.c b/hello_world2.c
new file mode 100644
index 0000000..2373725
--- /dev/null
+++ b/hello_world2.c
@@ -0,0 +1,6 @@
+#include<stdio.h>
+int main(){
+	printf("Hello World!!\n");
+	printf("Hello World 2!!\n");
+	printf("Hello World 3!!\n");
+}
-- 
2.25.1

