From e5f4e344b48d4b400b9a76d935fc05bcb1e0b6e5 Mon Sep 17 00:00:00 2001
From: Aryan Sahoo <sahooaryan499@gmail.com>
Date: Wed, 31 Jan 2024 12:48:08 +0530
Subject: [PATCH 2/2] hello_world3.c commit

---
 hello_world3.c | 7 +++++++
 1 file changed, 7 insertions(+)
 create mode 100644 hello_world3.c

diff --git a/hello_world3.c b/hello_world3.c
new file mode 100644
index 0000000..1dc83d4
--- /dev/null
+++ b/hello_world3.c
@@ -0,0 +1,7 @@
+#include<stdio.h>
+int main(){
+	printf("Hello World!!\n");
+	printf("Hello World 2!!\n");
+	printf("Hello World 3!!\n");
+	printf("Hello World 4!!\n");
+}
-- 
2.25.1

